src/main/java/eu/execom/pomodoroTeam/entities/TeamEntity.java

```diff
@@ -24,7 +23,9 @@
     @Column(nullable = false)
     private String name;
 
-    @ManyToMany(mappedBy = "teams")
-    private List<UserEntity> users = new ArrayList<>();
-
+    @ManyToMany(cascade = CascadeType.REFRESH)
+    @JoinTable(name = "User_Team", joinColumns = {
+            @JoinColumn(name = "Team_id", nullable = false, updatable = false)}, inverseJoinColumns = {
+            @JoinColumn(name = "User_id", nullable = false, updatable = false)})
+    private List<UserEntity> user = new ArrayList<>();

```

Comment:
Please pay attention to naming - there are multiple users, therefore the field should be named `users`.

Reply:
This is still not fixed -> please rename `user` to `users`.